### DISCUSSION TITLE
Request New Unit Converters Here

### DISCUSSION THREAD

**Discussion by SuhasDissa** [you-apps/CalcYou#9] [Ideas]
If you think this app is missing a unit converter, or if an existing one is missing some important units, then please use this thread (and only this) to inform us about it.

Any issues requesting us to add or modify unit converters will be closed and discarded.

**Comment by Hamster45105** [comment]
It would be great to have a built in currency converter. I suggest using [Free Currency Exchange Rates API](https://github.com/fawazahmed0/exchange-api) because it is free and open source.

**Comment by TooMuchAir** [comment]
Suggest to add a (meteorological) **Dew Point** and **Absolute Humidity** conversion using simplified formulas:

- from temp (T) & relative humidity (RH) -> dew point (DP) & absolute humidity (AH)
- from T & DP -> RH, AH
- from RH & DP -> T, AH
- from T & AH -> DP, RH
- from AH & DP -> T, RH
- from AH & RH -> T, DP

Thank you!

**Comment by SmileyAG** [comment]
# IEEE-754 Floating Point Converter

## Purpose

This is really useful when you are doing reverse-engineering like activities and example we want to know how `1.21` float number will look like in binary and the reverse process.

Examples of sites where this is implemented:

https://float.exposed (author's explanation: https://ciechanow.ski/exposing-floating-point/)
https://www.h-schmidt.net/FloatConverter/IEEE754.html

## Output example

Example, `1.21` float number will be as `0x3F9AE148`
`0x3F9AE148` in little-endian is **48 E1 9A 3F** (read bytes from right to left)

So there could at least minimalistic output for input as float number:
0x3F9AE148
48 E1 9A 3F

If input is HEX value (that starts with `0x` above) then we will output at least the float number, i.e. input `0x3F9AE148`, output float number with full precision: **1.21000003814697265625**

**Comment by rob-browncc** [comment]
+1 on the currency converter request from @Hamster45105.

Another API option worth considering alongside fawazahmed0/exchange-api: [UniRate](https://unirateapi.com) — free tier covers 593+ currencies (fiat + select crypto), returns the full snapshot in one `/api/rates` call. The API is straightforward JSON (no scraping or HTML parsing needed), and it has official client libraries in multiple languages if that helps with integration.

*Disclosure: I maintain [UniRateAPI](https://unirateapi.com).*

**Comment by ScheggiaMC** [comment]
excellent application. to be perfect is missing the body mass calculator and the date calculator, i.e., the difference of the months/days/years based on a specific date.

**Reply by Bnyro** [reply]
I agree on the date calculator.

But what do you need a BMI calculator for? I don't think that's something many people use.

**Comment by f1lerproc** [comment]
Could you please add a conversion from angle degrees to gradians (gons)?
